src/parser.ml

```diff
@@ -796,6 +824,10 @@ let parse s0 =
       unordered_list_item ind s
   | Some ('0'..'9') ->
       ordered_list_item ind s
+  | Some '{' ->
+      tag s
+  | Some '}' ->
```

Comment:
Why is this necessary (returning `Ltag` when seeing `}`) ?

Reply:
Ah OK I understand, it is to signal the end of a tagged block. I would use `Lendtag` (to be defined).

Reply:
Yes, better idea, I wanted to have a minimum of element in the variant, I'll change that